lib/os.js

```diff
@@ -89,16 +89,16 @@ function cpus() {
   // [] is a bugfix for a regression introduced in 51cea61
   const data = getCPUs() || [];
   const result = [];
-  for (var i = 0; i < data.length; i += 7) {
+  for (var i = 0, j = 0; i < data.length; i++) {
     result.push({
-      model: data[i],
-      speed: data[i + 1],
+      model: data[j++],
+      speed: data[j++],
       times: {
-        user: data[i + 2],
-        nice: data[i + 3],
-        sys: data[i + 4],
-        idle: data[i + 5],
-        irq: data[i + 6]
+        user: data[j++],
+        nice: data[j++],
+        sys: data[j++],
+        idle: data[j++],
+        irq: data[j++]
       }
     });
```

Comment:
This is not identical to the former. `i` is not necessary anymore and causes the wrong output.

The loop could either be changed to `for (var j = 0; j < data.length;) { ... }` (I would keep `i` instead of `j`) or `let i = 0;\nwhile (i < data.length) { ... }`

Reply:
@ZYSzys @BridgeAR - thanks, fixed now, ptal.